articles/machine-learning/how-to-prepare-datasets-for-automl-images.md

```diff
@@ -50,6 +50,10 @@ If you already have a data labeling project and you want to use that data, you c
 ### Using pre-labeled training data
```

Comment:
Need similar changes for v1 version of the doc as well.

Reply:
I'll create a seperate PR for v1